src/pages/components/file-upload.mdx

```diff
@@ -123,10 +123,26 @@ The FileUpload component follows this structure:
 <Component components={props.data.mdx.components} componentName="field" />
 
 A Field provides accessibility attributes to its child [FileUpload](#fileupload) field by associating it
-with the corresponding [Label](#label) and [Hint](#hint).
+with the corresponding [Field.Label](#fieldlabel) and [Field.Hint](#fieldhint).
 
 <PropSheet components={props.data.mdx.components} componentName="field" />
 
+#### Field.Hint
+
+<Component components={props.data.mdx.subcomponents} componentName="field.hint" />
+
+Nest the Hint within a [Field](#field) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="field.hint" />
+
+#### Field.Label
+
+<Component components={props.data.mdx.subcomponents} componentName="field.label" />
+
+Nest the Label within a [Field](#field) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="field.label" />
+
```

Comment:
Should this page include `Field.Message` for consistency with the other `forms` pages?

Reply:
It probably should.